### PR TITLE
Add information for Raspberry Pi Image

### PR DESCRIPTION
**Pull request by Landrash**
Feedback welcome and feel free to edit anything you feel needs to be changed.

### CODE REVIEW

File: source/getting-started/installation-raspberry-pi-image.markdown [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+On first boot the latest release of Home Assistant will be installed in a [Virtual Environment](getting-started/installation-virtualenv) at `/srv/homeassistant` and started as a service run by the user `homeassistant`. Since this is done automaticly it's required that the Raspberry Pi is connected to the internet with a ethernet cable. Installation takes about 5 minutes on first boot and after that you can connect to your Raspberry pi on `ipaddress:8123`.
+
+Pre-installed on this image is the MQTT broker [Mosquitto](https://mosquitto.org/), Bluetooth support for none Bluetooth LE and settings for the `homeassistant` user to use the GPIO pins of the Raspberry Pi. Mosquitto is not activated by default.
+
+As it is today there are no pre-compiled Z-Wave support but it can be installed by following the [Getting started instructions for Z-Wave](/getting-started/z-wave/).
+
+For Raspberry Pi hardware specific components have a look at our [install guide for the Raspberry Pi](/getting-started/installation-raspberry-pi/#raspberry-pi-hardware-specific-components). Default password for the pi` user is `raspberry` and is highly recommended to be changed.
```
Comment (MartinHjelmare) [inline source/getting-started/installation-raspberry-pi-image.markdown]: Remove the extra backtic after pi.

File: source/getting-started/installation-raspberry-pi.markdown [hunk 2]
```diff
@@ -11,6 +11,14 @@ footer: true
 
 ### {% linkable_title Installation %}
 
+There's currently three documented ways to install Home Assistant on a Raspberry Pi.  
+ - [Manual installation](/getting-started/installation-raspberry-pi/#Manual-Installation). Following this guide doing each step manualy. This is highly recommended as a first installtion since you get a good overview of the installation.
+ - [Hassbian image](/getting-started/installation-raspberry-pi-image). Basic installation with the same settings as following the manual installtion guide. Some additional software is preinstalled to make installtion quicker and easier.
```
Comment (MartinHjelmare) [inline source/getting-started/installation-raspberry-pi.markdown]: the manual installation guide... to make installation quicker...

File: source/getting-started/installation-raspberry-pi-image.markdown (hunk 1 above)
Comment (Landrash) [inline source/getting-started/installation-raspberry-pi-image.markdown]: Or add an extra before :grin:

**Comment by Landrash** [comment]
@MartinHjelmare I greatly appreciate your feedback and think I've addressed all of it now.

File: source/getting-started/installation-raspberry-pi.markdown [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: page
-title: "Raspbian Lite on a Raspberry Pi"
+title: "Manual installation a Raspberry Pi"
```
Comment (MartinHjelmare) [inline source/getting-started/installation-raspberry-pi.markdown]: Manual installation on a Raspberry Pi

File: source/getting-started/installation-raspberry-pi.markdown [hunk 4]
```diff
@@ -11,6 +11,14 @@ footer: true
 
 ### {% linkable_title Installation %}
 
+There's currently three documented ways to install Home Assistant on a Raspberry Pi.  
+ - [Manual installation](/getting-started/installation-raspberry-pi/#Manual-Installation). Following this guide doing each step manualy. This is highly recommended as a first installtion since you get a good overview of the installation.
```
Comment (MartinHjelmare) [inline source/getting-started/installation-raspberry-pi.markdown]: doing each step manually... as a first installation...

**Comment by Landrash** [comment]
@MartinHjelmare They should be addressed.
@fabaff If you don't mind I would love for you to also have a look over this or at the minimum just confirm the to `source/_includes/asides/getting_started_navigation.html`. Everything renders properly as is for me but I'm not that familiar with Jekyll to be sure that I didn't break something else.

**Pull request merged**